etc/picongpu/hemera-hzdr/k80_picongpu.profile.example

```diff
@@ -0,0 +1,85 @@
+# Name and Path of this Script ############################### (DO NOT change!)
+export PIC_PROFILE=$(cd $(dirname $BASH_SOURCE) && pwd)"/"$(basename $BASH_SOURCE)
+
+# User Information ######################################### (edit those lines)
+#   - automatically add your name and contact to output file meta data
+#   - send me a mail on batch system jobs: NONE, BEGIN, END, FAIL, REQUEUE, ALL,
+#     TIME_LIMIT, TIME_LIMIT_90, TIME_LIMIT_80 and/or TIME_LIMIT_50
+export MY_MAILNOTIFY="NONE"
+export MY_MAIL="someone@example.com"
+export MY_NAME="$(whoami) <$MY_MAIL>"
+
+# Text Editor for Tools ###################################### (edit this line)
+#   - examples: "nano", "vim", "emacs -nw", "vi" or without terminal: "gedit"
+#export EDITOR="nano"
+
+# General modules #############################################################
+#
+module purge
+module load gcc/7.3.0
+module load cmake/3.11.3
+module load cuda/10.0
+module load openmpi/2.1.2-cuda100
+module load boost/1.68.0
+
+# Other Software ##############################################################
+#
+module load zlib/1.2.11
+module load c-blosc/1.14.4
+
+module load adios/1.13.1-cuda100
+module load hdf5-parallel/1.8.20-cuda100
+module load libsplash/1.7.0-cuda100
+
+export CMAKE_PREFIX_PATH=$CMAKE_PREFIX_PATH:$SPLASH_ROOT:$HDF5_ROOT
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$HDF5_ROOT/lib
+
+module load libpng/1.6.35
+module load pngwriter/0.7.0
+
+# Environment #################################################################
+#
+#export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$BOOST_LIB
+
+export PICSRC=$HOME/src/picongpu
+export PIC_EXAMPLES=$PICSRC/share/picongpu/examples
+export PIC_BACKEND="cuda:37"
+
+export PATH=$PATH:$PICSRC
+export PATH=$PATH:$PICSRC/bin
+export PATH=$PATH:$PICSRC/src/tools/bin
+
+export PYTHONPATH=$PICSRC/lib/python:$PYTHONPATH
+
+# "tbg" default options #######################################################
+#   - SLURM (sbatch)
+#   - "gpu" queue
+export TBG_SUBMIT="sbatch"
+export TBG_TPLFILE="etc/picongpu/hemera-hzdr/k80.tpl"
+
+# allocate an interactive shell for one hour
+#   getNode 2  # allocates to interactive nodes (default: 1)
```

Comment:
`getNode` didn't work. The error message is `srun: error: Unable to allocate resources: Requested node configuration is not available`. I change the allocation from gpu to k80, still didn't work

Reply:
Thanks for reporting that :+1: I'll have a look

Try `salloc -p k80 -N1 -J k80_interactive -t 2:00:00 -A k80`.
You will also get your job. Just look at `squeue -u $USER`, find out your node and `ssh` to it.
Then you can also build there.

Reply:
@psychocoderHPC didn't we recently start to change that function for slurm? If I remember correctly, our plan was to `salloc` in the `getNode` function and `echo` the `srun` instructions to the user.

Reply:
Yes we changed it on the jülich systems to use salloc but since `srun` on hemera is still not working we can keep it.
If we change it to `salloc` you must call `srun mpiexec ...` to start your mpi environment which is very ugly and non standard.

Reply:
It does seem to work now. I had just forgotten the `-A $proj`

Reply:
Can I resolve this conversation since both the `getNode` and the `getDevice` function work in the current configuration?

Reply:
I will mark it as resolved since and we can change it as soon as our cluster supports srun correctly.
The admin is aware about this broken srun.